site/_posts/2021-07-03-avoid-lockin-insulate-your-application-and-have-backup-plan-excerpt.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog-post
+title: Avoid Lock-In by Insulating Your Application and Having a Backup Plan
+description: Identiverse is an annual identity and IAM conference. FusionAuth showed up and took notes so you don't have to.
+author: Cameron Pavey
+image: blogs/takeaways-identiverse-2021/takeaways-from-identiverse-2021-covid-thoughts-fido-and-more-header-image.png
+category: blog
+tags: conference-report
+excerpt_separator: "<!--more-->"
+---
+
+Years ago your team decided to use a third-party authentication system to avoid the time and cost of building one in-house. But now a better option has hit the market and you're wanting to make the switch. Except, hold on, your old system is so deeply ingrained into your organization that you're practically locked-in to your current vendor.
+
+<!--more-->
+
+How can you avoid this? 
+
+Two strategies that work are:
+
+* Insulate Your Application
+* Have a Backup Plan
+
+_This blog post is an excerpt from [Avoiding Authentication System Lock-in](/learn/expert-advice/identity-basics/avoid-lockin/)._
+
+## Insulate your application
```

Comment:
"Isolate your application"?

Reply:
Insulate is a better fit. "insulate your application from the API of your auth provider" makes more sense than "isolate your application from the auth provider" to me.